.github/workflows/docker.yaml

```diff
@@ -0,0 +1,24 @@
+name: Release and version
```

Comment:
Any reason we're not pushing to the `ghcr.io` registry?

Example of a release workflow for GHCR, used for our `build` images: https://github.com/ethereum-optimism/kona/blob/main/.github/workflows/cross_docker.yaml